FreeRTOS/FreeRTOS-Partner-Supported-Demos
# Add demo project for Renesas F1Kx CC-RH

Add Renesas F1KX CC-RH FreeRTOS Demo

Description
-----------
This directory contains two demo projects for the RH850 F1Kx using the CC-RH compiler and running the FreeRTOS kernel. The projects are named `SMP_SingleCore` and `SMP_MultiCore`.
- `SMP_SingleCore`: Demo project for a single-core setup.
- `SMP_MultiCore`: Demo project for a multi-core setup.

Test Steps
-----------
The testing is following the template in [README.md](https://github.com/FreeRTOS/FreeRTOS/blob/main/FreeRTOS/Demo/ThirdParty/Template/README.md)

Related Issue
-----------
None
No changes outside Demo directory (FreeRTOS-Partner-Supported-Demos/).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
